CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+### Added
+
+- Added `IpAddress` GraphQL scalar for IP addresses.
```

Comment:
From the user's perspective, they would want to know the constraints this type enforces for validation, as well as the errors the API returns when a value doesn't meet those constraints. Therefore, I think it would be better to mention that the API returns a specific error message if the value cannot be parsed into `IpAddr`.

Reply:
The correct term is "custom scalar," not just "scalar".

Reply:
To save your time, I let you know the error message when "abc" is input:

`Failed to parse "IpAddress": Invalid IP address: abc (occurred while parsing "[IpAddress!]")`